internal/intent/intent.go

```diff
@@ -0,0 +1,86 @@
+package intent
+
+import "fmt"
+
+const (
+	TypeReleaseArtifactID = "ReleaseArtifactID"
+	TypeReleaseBranch     = "ReleaseBranch"
+	TypePromote           = "Promote"
+	TypeRollback          = "Rollback"
+	TypeAutoRelease       = "AutoRelease"
+)
+
+type Intent struct {
+	Type          string              `json:"intentType,omitempty"`
```

Comment:
Is this deliberately using another JSON field name?

Reply:
no... Fixing! 👍